components/playlist/playlist_service.cc

```diff
@@ -423,6 +424,16 @@ std::vector<PlaylistInfo> PlaylistService::GetAllPlaylists() {
   return result;
 }
 
+void PlaylistService::FindMediaFilesFromContents(
```

Comment:
Seems this method is not used now.
Will be used in the future only for finding media files list?

Reply:
Yes, we're likely to have a button on the location bar to trigger this.